ynr/apps/elections/uk/templates/official_documents/_post.html

```diff
@@ -17,7 +17,7 @@
           adding them to this google spreadsheet</a>?
         {% endif %}
         {% if user_can_upload_documents %}
-          {% if election == '2015' %}
+          {% if election.slug == '2015' %}
```

Comment:
I'm confused by this. If we're hard-coding an ID, It seems like this should be `parl.2015-05-07`, but I checked https://github.com/DemocracyClub/yournextrepresentative/pull/729 and we're not updating it. Are we making the mistake of only updating legacy ballot ids, but not the corresponding legacy election ids here? If we've still got legacy ids in place, we probably need to fix those before we deploy this and expose them.

Reply:
Right, yep. Don't mind if its most logical to fit it in here or another PR - as long as its on your radar of stuff that needs fixing under this umbrella :+1: